site/content/docs/commands/project.md

```diff
@@ -7,18 +7,19 @@
 ### Synopsis
 
 ```
-dotnet aws deployment-project generate [-d|--diagnostics] [-s|--silent] [--profile <PROFILE>] [--region <REGION>] [--project-path <PROJECT-PATH>] [--project-display-name <DISPLAY-NAME>]
+dotnet aws deployment-project generate [-o|--output <OUTPUT-PATH>] [-d|--diagnostics] [--project-path <PROJECT-PATH>] [--project-display-name <DISPLAY-NAME>]
 ```
 
 ### Description
-Generates and saves the [deployment CDK project](../deployment-projects/index.md) in a user provided directory path without proceeding with a deployment. Allows user to customize the CDK project before deploying the application.
+Generates and saves the [deployment CDK project](../deployment-projects/index.md) in a user-provided directory without proceeding with a deployment. This allows the user to customize the CDK project before deploying the application.
 
-* The `--output` switch sets the directory where the deployment project will be saved.
-* The `--project-display-name` switch sets the name that will be shown when the .NET project is being deployed.
+* The `--output` switch sets the directory where the CDK deployment project will be saved.
+* The `--project-path` switch sets the path to the project to deploy.
```

Comment:
replace `the project to deploy` to `the project to generate a deployment project for` since we aren't deploying the project.

Reply:
Agreed, updated with your suggestion.

I had originally just copy+pasted what we had here to match the CLI's `--help`, but looks like we're using a shared string there across all commands. It could probably be improved, but would rather keep this PR focused on the docs site.
https://github.com/aws/aws-dotnet-deploy/blob/3bc852c68eaad581ab92a12cae8c6d93625e72ee/src/AWS.Deploy.CLI/Commands/CommandFactory.cs#L42